packages/lumx-react/src/components/thumbnail/Thumbnail.tsx

```diff
@@ -139,7 +139,23 @@ const Thumbnail: React.FC<ThumbnailProps> = ({
     focusPoint = DEFAULT_PROPS.focusPoint,
     ...props
 }: ThumbnailProps): ReactElement => {
-    const focusImageRef = useFocusedImage(focusPoint!);
+    const focusImageRef = useFocusedImage(focusPoint!, aspectRatio!, size!);
+
+    /**
+     * Check if browser is IE
+     * @return Browser is IE or not
+     */
+    const isInternetExplorer = () => {
```

Comment:
It could be very useful to have this function standalone in a util file. 
Something like: `src/utils/isInternetExplorer.ts`